docs/dataframe_api.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
<p>Suggest to add titles to these plots.  Left is "accuracy", Right is "loss"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done adding titles.</p>